webpack/test-utils/findForeman.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const fs = require('fs');
+
+const getForemanLocation = () => {
```

Comment:
Consider dropping `get` from the names `getForemanLocation` and `getForemanRelativePath`.  That way the function names reference what they return, rather than the task they perform.